### PR TITLE
Refactored the code

### PR DESCRIPTION
**Pull request by uday-krishnappa**

### CODE REVIEW

File: integer-partition.py [hunk 1]
```diff
@@ -1,9 +1,3 @@
-
```
Comment (ManuAc) [inline integer-partition.py:1]: why this change ??

**Review by ManuAc — CHANGES_REQUESTED** [review]

File: integer-partition.py (hunk 1 above)
Comment (uday-krishnappa) [reply integer-partition.py:1]: Sir, because you asked me to.

**Review by ManuAc — APPROVED** [review]
